sdk/spring/spring-cloud-azure-feature-management/pom.xml

```diff
@@ -11,7 +11,7 @@
 
   <groupId>com.azure.spring</groupId>
   <artifactId>spring-cloud-azure-feature-management</artifactId>
-  <version>4.0.0-beta.4</version> <!-- {x-version-update;com.azure.spring:spring-cloud-azure-feature-management;current} -->
+  <version>4.7.0-beta.1</version> <!-- {x-version-update;com.azure.spring:spring-cloud-azure-feature-management;current} -->
```

Comment:
https://github.com/Azure/azure-sdk-for-java/pull/34169 this needs to be merged before release.